lib/routes.js

```diff
@@ -113,6 +113,12 @@ publicRoutes.route('/posts/:postId', {
   }
 });
 
+publicRoutes.route('/kontakt', {
+ action: () => {
+   BlazeLayout.render('kontakt');
```

Comment:
You are probably right. FlowRouter is new to me. 

> 19. mai 2016 kl. 15.42 skrev Sebastian Sastre <notifications@github.com>:
> 
> In lib/routes.js <https://github.com/dagvonkr/hof3/pull/11#discussion_r63879961>:
> 
> > @@ -113,6 +113,12 @@ publicRoutes.route('/posts/:postId', {
> >    }
> >  });
> >  
> > +publicRoutes.route('/kontakt', {
> > + action: () => {
> > +   BlazeLayout.render('kontakt');
> Are you sure you want to render it directly without using any layout? like the blogLayout (which adds the blog header) @dagvonkr <https://github.com/dagvonkr> ?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/dagvonkr/hof3/pull/11/files/f1a0b514e1ddbd8dbcd90b014e189a7cdea7bcb6#r63879961>